src/contracts/pods/EigenPod.sol

```diff
@@ -205,6 +211,14 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
             "EigenPod.verifyBalanceUpdate: specified timestamp is too far in past"
         );
 
+        /**
+        * checks that a balance update can only be made before the validator is withdrawable.  If this is not checked
+        * anyone can prove the withdrawn validator's balance as 0 before the validator is able to prove their full withdrawal
+        */
+        require(Endian.fromLittleEndianUint64(validatorFields[BeaconChainProofs.VALIDATOR_WITHDRAWABLE_EPOCH_INDEX]) >
```

Comment:
this should also check that the balance == 0. it could be that the validator is withdrawable but not withdrawn yet. pls update revert message accordingly

Reply:
fixed, good catch